supervisord.conf

```diff
@@ -12,8 +12,56 @@ autorestart = true
 command = /bin/bash /startup.sh
 priority = 1
 
-[program:carbon-cache]
-command = /var/lib/graphite/bin/carbon-cache.py --debug start
+[program:carbon-cache-1]
+command = /var/lib/graphite/bin/carbon-cache.py --debug --instance=1 start
+stdout_logfile = /var/log/supervisor/%(program_name)s.log
+stderr_logfile = /var/log/supervisor/%(program_name)s.log
+autorestart = true
+
+[program:carbon-cache-2]
+command = /var/lib/graphite/bin/carbon-cache.py --debug --instance=2 start
+stdout_logfile = /var/log/supervisor/%(program_name)s.log
+stderr_logfile = /var/log/supervisor/%(program_name)s.log
+autorestart = true
+
+[program:carbon-cache-3]
+command = /var/lib/graphite/bin/carbon-cache.py --debug --instance=3 start
+stdout_logfile = /var/log/supervisor/%(program_name)s.log
+stderr_logfile = /var/log/supervisor/%(program_name)s.log
+autorestart = true
+
+[program:carbon-cache-4]
+command = /var/lib/graphite/bin/carbon-cache.py --debug --instance=4 start
+stdout_logfile = /var/log/supervisor/%(program_name)s.log
+stderr_logfile = /var/log/supervisor/%(program_name)s.log
+autorestart = true
+
+[program:carbon-cache-5]
+command = /var/lib/graphite/bin/carbon-cache.py --debug --instance=5 start
+stdout_logfile = /var/log/supervisor/%(program_name)s.log
+stderr_logfile = /var/log/supervisor/%(program_name)s.log
+autorestart = true
+
+[program:carbon-cache-6]
+command = /var/lib/graphite/bin/carbon-cache.py --debug --instance=6 start
+stdout_logfile = /var/log/supervisor/%(program_name)s.log
+stderr_logfile = /var/log/supervisor/%(program_name)s.log
+autorestart = true
+
+[program:carbon-cache-7]
+command = /var/lib/graphite/bin/carbon-cache.py --debug --instance=7 start
+stdout_logfile = /var/log/supervisor/%(program_name)s.log
+stderr_logfile = /var/log/supervisor/%(program_name)s.log
+autorestart = true
+
+[program:carbon-cache-8]
+command = /var/lib/graphite/bin/carbon-cache.py --debug --instance=8 start
+stdout_logfile = /var/log/supervisor/%(program_name)s.log
+stderr_logfile = /var/log/supervisor/%(program_name)s.log
+autorestart = true
+
+[program:carbon-c-relay]
+command = carbon-c-relay -w 8 -f /var/lib/graphite/conf/carbon-c-relay.conf start
```

Comment:
nit: if the `-w 8` stands for worker or has a long param name, we should prefer that here imo, to make it clear to the reader what the 8 means

Reply:
I am afraid there are no long parameter names in https://github.com/grobian/carbon-c-relay